src/software/jetson_nano/setup_nano.sh

```diff
@@ -34,7 +34,7 @@ pip_libaries=(
     adafruit-circuitpython-rgb-display==3.10.9
     Adafruit-PlatformDetect==3.18.0
     Adafruit-PureIO==1.1.9
-    Jetson.GPIO==2.0.17
+    Jetson.Gpio==2.0.17
```

Comment:
python imports are case sensitive (https://pypi.org/project/Jetson.GPIO/) so this should still be Jetson.GPIO